### PR TITLE
filled out .janno

### PR DESCRIPTION
**Pull request by Kavlahkaff**
Started to fill the .janno file. The CollectionId is not yet added, as I was unsure whether to use the ANTPA catalogue id or the Machu Picchu cemetery number.

### CODE REVIEW

**Comment by stschiff** [comment]
Great, thanks @Kavlahkaff. I think CollectionID is a list column, so you should just put both.

**Comment by Kavlahkaff** [comment]
In the [schema]( https://github.com/poseidon-framework/poseidon-schema/blob/master/janno_columns.tsv) it is marked as single entry. Should I put both nonetheless or only one of them?

**Comment by nevrome** [comment]
I agree that it should be a list column. Let's change that with the next schema release. Could you please open a one-sentence [issue](https://github.com/poseidon-framework/poseidon-schema/issues) about this in the schema repo, @Kavlahkaff?

For the moment the column just stores a string, so I think you can add a list `x;y;...` without breaking anything.

**Comment by Kavlahkaff** [comment]
Okay, I opened an [issue](https://github.com/poseidon-framework/poseidon-schema/issues/84) in the Poseidon schema repo. I will then add the values as you proposed.

**Comment by Kavlahkaff** [comment]
I have now added and commited the CollectionID's to the individuals where I found corresponding values.

File: 2023_Salazar_MachuPicchu/2023_Salazar_MachuPicchu.janno [hunk 1]
```diff
@@ -1,57 +1,57 @@
...
+SHN-902_MNT	F	Cusco City;Cusco;Sacsahuaman	n/a	n/a	n/a	n/a	n/a	n/a	Peru	PE	Cusco City;Cusco	Sacsahuaman	-13,50986	-71,98169	contextual	n/a	n/a	n/a	1400	1466	1533	Dates (CE) taken from Table S1 A	D1h	n/a	Tooth (molar)	1	SHN-902	1240K	half	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.4.6	n/a	327344	n/a	7,06E+12	185.285	6,38E-08	ANGSD[v0.935]	Nr Snps (per library): 916. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB62808;ERS15565588;ERR11518584	n/a	n/a	n/a	n/a	SHN-902	SHN-902	6.839.695.315.628	7.178.577.511.500.000	7.962.545.374.461.060	5.962.469.648.282.000
```
Comment (TCLamnidis) [inline 2023_Salazar_MachuPicchu/2023_Salazar_MachuPicchu.janno:57]:
```suggestion
SHN-902_MNT	F	Cusco City;Cusco;Sacsahuaman	n/a	n/a	n/a	n/a	n/a	n/a	Peru	PE	Cusco City;Cusco	Sacsahuaman	-13,50986	-71,98169	contextual	n/a	n/a	n/a	1400	1466	1533	Dates (CE) taken from Table S1 A	D1h	n/a	Tooth (molar)	1	SHN-902	1240K	half	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.4.6	n/a	327344	n/a	7,06E+12	185.285	6,38E-08	ANGSD[v0.935]	Nr Snps (per library): 916. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB62808;ERS15565588;ERR11518584	n/a	n/a	n/a	n/a	SHN-902	SHN-902	6.839.695.315.628	7.178.577.511.500.000	7.962.545.374.461.060	5.962.469.648.282.000

```

**Review by TCLamnidis — CHANGES_REQUESTED** [review]
The added information looks great, BUT:
Please do NOT replace any information that was already filled by MINOTAUR (e.g. Nr_SNPs, RateX/Y etc). This is information that is dependent on the data processing, and should therefore reflect our processing, and not the author's.

Please revert the changes to those columns. The rest looks great!

File: 2023_Salazar_MachuPicchu/2023_Salazar_MachuPicchu.janno (hunk 1 above)
Comment (Kavlahkaff) [reply 2023_Salazar_MachuPicchu/2023_Salazar_MachuPicchu.janno]: Hi, I reverted the changes made to those columns. I am not sure exactly how they happened, as I did not change them intentionally. Might have happened while changing between file formats.

File: 2023_Salazar_MachuPicchu/2023_Salazar_MachuPicchu.janno [hunk 2]
```diff
@@ -1,57 +1,57 @@
...
+SHN-902_MNT	F	Cusco City;Cusco;Sacsahuaman	n/a	n/a	n/a	n/a	n/a	n/a	Peru	PE	Cusco City;Cusco	Sacsahuaman	-13.50986	-71.98169	contextual	n/a	n/a	n/a	1400	1466	1533	Dates (CE) taken from Table S1 A	D1h	n/a	Tooth (molar)	1	SHN-902	1240K	half	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.4.6	n/a	327344	n/a	7.058316148176045e-3	0.185285	6.380789e-14	ANGSD[v0.935]	Nr Snps (per library): 916. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB62808;ERS15565588;ERR11518584	n/a	n/a	n/a	n/a	SHN-902	SHN-902	0.006839695315628	0.0007178577511500001	0.7962545374461061	0.005962469648282001
```
Comment (TCLamnidis) [inline 2023_Salazar_MachuPicchu/2023_Salazar_MachuPicchu.janno:57]:
```suggestion
SHN-902_MNT	F	Cusco City;Cusco;Sacsahuaman	n/a	n/a	n/a	n/a	n/a	n/a	Peru	PE	Cusco City;Cusco	Sacsahuaman	-13.50986	-71.98169	contextual	n/a	n/a	n/a	1400	1466	1533	Dates (CE) taken from Table S1 A	D1h	n/a	Tooth (molar)	1	SHN-902	1240K	half	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.4.6	n/a	327344	n/a	7.058316148176045e-3	0.185285	6.380789e-14	ANGSD[v0.935]	Nr Snps (per library): 916. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB62808;ERS15565588;ERR11518584	n/a	n/a	n/a	n/a	SHN-902	SHN-902	0.006839695315628	0.0007178577511500001	0.7962545374461061	0.005962469648282001

```

**Review by TCLamnidis — CHANGES_REQUESTED** [review]
- [ ] An additional file has been committed that does not belong to this package: `2023_Ferraz_NatureEcologyEvolution/2023_Ferraz_NatureEcologyEvolution2.csv`
- [ ] `2023_Salazar_MachuPicchu/2023_Salazar_MachuPicchu.janno` needs a new line at EOF.

**Review by TCLamnidis — APPROVED** [review]
Very nice @Kavlahkaff ! One last thing:

- [x] `Tooth (molar)` in the `Source_Tissue` should be replaced with `tooth_molar` as per the example in the website.

**Pull request merged**